.pipelines/containerSourceData/scripts/PublishContainers.sh

```diff
@@ -336,6 +338,21 @@ do
                 "$major_version" \
                 "$azure_linux_version" \
                 "$ARCHITECTURE_TO_BUILD"
+        elif "$IS_NVIDIA_GOLDEN_IMAGE"; then
+            # For nvidia images, we need to create multi-arch tag for the following format:
+            # E.g. for azurelinuxpreview.azurecr.io/nvidia/cuda/driver:550-5.15.160.1-1.cm2-mariner2.0.2.0.20240626-amd64
+            # the multi-arch tag to create is: azurelinuxpreview.azurecr.io/nvidia/cuda/driver:550-5.15.160.1-1.cm2-mariner2.0
+            multiarch_tag=${container_tag%.*.*.*}
+
+            azure_linux_version=$(awk -F '-' '{print $4}' <<< "$container_tag")              # 550-5.15.160.1-1.cm2-mariner2.0.2.0.20240626 -> mariner2.0.2.0.20240626
+            azure_linux_version=$(awk -F '.' '{print $3"."$4}' <<< "$azure_linux_version")   # [mariner2].[0].[2].[0].[20240626] -> 2.0
+            #                                                                                                  ^   ^
```

Comment:
Was there a comment here

Reply:
Yes, this line is a comment that includes the two "^" on the right.